pom.xml

```diff
@@ -83,6 +84,11 @@
             <artifactId>structs</artifactId>
             <version>1.5</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>script-security</artifactId>
+            <version>1.29</version>
```

Comment:
This is the most recent `script-security` release that doesn't use a newer version of core than we do here.

We might also want to think about moving `@Whitelisted` to a separate plugin that just has the annotation, so that we don't need to pull in all of `script-security`'s dependencies here.

Reply:
`script-security` is considered a lightweight dependency—in fact it is in the detached list for a long time now, for `matrix-project`.